components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml

```diff
@@ -61,6 +61,36 @@ $xwiki.jsfx.use('js/scriptaculous/dragdrop.js')##
 
 #end
 ##
+## Tabs to be displayed after the family tab
+#set ($tabs = [])
+(% class="hidden" %)(((
+#foreach ($e in $services.uix.getExtensions('org.phenotips.entitySheet.tab', {'sortByParameter' : 'order'}))
+  #set ($extension_disabled = ($e.getParameters().get('enabled') == 'false'))
+  #if (!$extension_disabled)
+    #set ($label = $!{e.getParameters().get('label')})
+    #set ($href = $!{e.getParameters().get('href')})
+    #set ($cssClass = $!{e.getParameters().get('cssClass')})
+    #set ($container = $!{e.getParameters().get('container')})
```

Comment:
This `container` is only used to detect which of the tabs is currently being displayed, and so could be named a bit better (or a different current tab detection system can be used - e.g. based on the page URL?). But I can't come up with a better name and not sure what other method is easy to implement, so this is minor and OK